css-positioning/absolute.css

```diff
@@ -0,0 +1,43 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
```

Comment:
this element should be position relative so that it acts as a containing block for the absolutely position elements